tensorflow/lite/micro/tools/python_interpreter/src/tflm_runtime.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Python package for TFLM Python Interpreter"""
+
+import os
+
+from tflite_micro.tensorflow.lite.micro.tools.python_interpreter.src import interpreter_wrapper_pybind
+
+
+class Interpreter(object):
+
+  def __init__(self, model_data, arena_size):
+    self._interpreter = interpreter_wrapper_pybind.InterpreterWrapper(
+        model_data, arena_size)
+
+  @classmethod
+  def from_file(self, model_path, arena_size=100000):
+    if model_path is None or not os.path.isfile(model_path):
+      raise ValueError("Invalid model file path")
+
+    with open(model_path, "rb") as f:
+      model_data = f.read()
+
+    return Interpreter(model_data, arena_size)
+
+  @classmethod
+  def from_bytes(self, model_data, arena_size=100000):
+    if model_data is None:
+      raise ValueError("Model must not be None")
+    return Interpreter(model_data, arena_size)
+
+  def invoke(self):
+    return self._interpreter.Invoke()
+
+  def set_input(self, input_data, index):
```

Comment:
can you add doc on meaning of index? Is it the tensor index from the tflite model or is it the index defined by the first op?

Reply:
Done.